confgenerator/testdata/invalid/linux/logging-receiver_duplicated_port/input.yaml

```diff
@@ -0,0 +1,18 @@
+logging:
```

Comment:
Can we also add a unit test case like this for completeness?

```
logging:
  receivers:
    s1:
      type: syslog
      transport_protocol: tcp
      listen_host: 0.0.0.0
      listen_port: 5140      
    s2:
      type: syslog
      transport_protocol: tcp
      listen_host: 0.0.0.0
      listen_port: 5140  
  service:
    pipelines:
      p1:
        receivers: [s1, s2]
```

Reply:
Test case added as `logging-receiver_duplicated_port_same_service`